src/isar_exr/robotinterface.py

```diff
@@ -280,14 +281,26 @@ def step_status(self) -> StepStatus:
         return step_status
 
     def stop(self) -> None:
-        try:
-            self.api.pause_current_mission(self.exr_robot_id)
-        except Exception:
-            message: str = "Could not stop the running mission\n"
-            self.logger.error(message)
-            raise RobotCommunicationException(
-                error_description=message,
-            )
+        max_request_attempts: int = 10
+        stop_mission_attempts: int = 0
+        while stop_mission_attempts < max_request_attempts:
```

Comment:
Maybe a comment on why we need to try stopping so many times?

Reply:
I think I'll test soon without these retries and see how reliable it is now that we have more error handling. We might not need the retries.